apps/client/src/common/components/view-params-editor/types.ts

```diff
@@ -4,9 +4,13 @@ type BaseField = {
   description: string;
 };
 
-type OptionsField = { type: 'option'; values: Record<string, string> };
-type StringField = { type: 'string' };
-type BooleanField = { type: 'boolean' };
-type NumberField = { type: 'number' };
+type OptionsField = {
+  type: 'option';
+  values: Record<string, string>;
+  defaultValue?: string;
```

Comment:
Quick question: I am not sure I can think of a case for a select field without a default? am I missing something?

Reply:
It's optional because some selects (like the time one) have a default value. And others, like the operator one, don't have a default value.